gradlew

```diff
@@ -80,6 +80,19 @@ case "`uname`" in
     ;;
 esac
 
+
+# Loop in case we encounter an error.
```

Comment:
This loop should terminate with an error if the wrapper can't be downloaded. A windows version would also be necessary. The problem is that Windows won't have curl so some other workaround is needed.

Reply:
The simplest workaround: a simple Java class that can be compiled on demand that would take an URL as an argument and place it in the given location. We do assume javac is available.

Reply:
This  turned out to be pretty straightforward, I'm skipping things like proxy and async and retry and pause/resume for now. I don't know enough about Windows scripting to understand what I need to change, but hopefully somebody else can help me out. Shouldn't be more than two lines.